src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Queries/Get/GetDialogDto.cs

```diff
@@ -47,7 +47,6 @@ public sealed class GetDialogDialogTransmissionDto
     public Guid Id { get; set; }
     public DateTimeOffset CreatedAt { get; set; }
     public string? AuthorizationAttribute { get; set; }
-    public bool IsAuthorized { get; set; }
```

Comment:
Breaking roight? Should this be removed?

Reply:
Yeah, ServiceOwner should not have the IsAuthorized flag, that is for EU only